docs/getting_started.md

```diff
@@ -41,12 +41,18 @@ While the `serve` command runs, every change you save triggers a rebuild.
 The first build is the slowest. After that, assets are cached on disk and
 incremental builds are faster.
 
+## Building output served by another server
```

Comment:
This title is confusing, sounds like you are building with inputs from some other server. The option is also generally useful for other things.

Maybe just `Creating an output directory`?

Reply:
I was hoping to find a header that matched the users intent rather than the solution.

How about "Building output to be served by another server"

Reply:
That is just one use case for this flag though, and not necessarily the main one?

Reply:
How about the latest? I changed the heading to something more general and mentioned the particular use case in the body.